draft-ietf-webpush-protocol.xml

```diff
@@ -481,11 +481,11 @@ iChYuI3jMzt3ir20P8r_jgRR-dSuN182x7iB
         </figure>
         <figure>
           <preamble>
-            A 202 (Accepted) response indicates that the push message was accepted. A
-            URI for the push message resource that was created in response to the request
-            MUST be returned in the Location header field. The push service MUST also provide
-            a URI for the receipt subscription resource in a link relation of type
-            "urn:ietf:params:push:receipt".
+            When the push service accepts the message for delivery, it MUST return a
+            202 (Accepted) response. A URI for the push message resource that was created

```

Comment:
Yes - this section is specific to _5.1. Requesting Push Message Receipts_ so I thought the context was clear especially with the previous paragraph. Would it be clearer to state:

When the push services accepts a message for delivery **with confirmation**